services/QuillLMS/app/assets/stylesheets/shared/navbar.scss

```diff
@@ -951,7 +951,7 @@
           .hover {
             display: none;
           }
-          #social-studies-img, #interdisciplinary-science-img, #for-administrators-img, #ethical-ai-img {
+          #social-studies-img, #interdisciplinary-science-img, #for-administrators-img, #ethical-ai-img, #science-of-reading-and-writing-img {
```

Comment:
This list is beginning to grow long enough that I wonder if it isn't worth standardizing a shared class for this and using it for each of these elements.